scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -26,12 +26,11 @@
 sys.path.append(os.path.join(_SCRIPT_PATH, '.'))
 
 from common.data_loader import process_subject_tables
-from .process import set_column_map
```

Comment:
removed this to prevent error when running ./run_tests.sh -p scripts/us_census/acs5yr/subject_tables/

Reply:
It seems that this removal is causing the Github tests to fail.
Could you add it back?

Reply:
done. Thanks for pointing it out.